python/common/org/python/types/Str.java

```diff
@@ -708,10 +708,21 @@ private long toPositiveIndex(long index) {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "string.isalnum() -> bool\n" +
+                    "\n" +
+                    "Returns false when the string has atleast ONE character which is\n" +
```

Comment:
I took [`isalpha()` ](https://github.com/pybee/voc/blob/master/python/common/org/python/types/Str.java#L727) as a reference. Should I keep it or remove?